pkg/issuer/acme/dns/util/wait.go

```diff
@@ -33,8 +35,13 @@ var RecursiveNameservers = getNameservers(defaultResolvConf, defaultNameservers)
 var DNSTimeout = 10 * time.Second
 
 // getNameservers attempts to get systems nameservers before falling back to the defaults
-func getNameservers(path string, defaults []string) []string {
-	config, err := dns.ClientConfigFromFile(path)
+func getNameservers(resolvConf string, defaults []string) []string {
```

Comment:
Instead of modifying the signature of this function, we'd be best to either:

1) Modify this package (fairly substantially) so it's 'initialised' based on config passed to it, and then 'initialize' the package in our cmd/ package.

2) Modify our cmd/controller/app/... package to set `util.RecurisveNameservers` to the value passed on the CLI (probably the easiest, but we need to ensure this happens *before* our control loops begin)

Reply:
Ultimately, this file shouldn't need to be changed at all in the end 😄